crates/uv/tests/edit.rs

```diff
@@ -3530,7 +3530,7 @@ fn add_virtual() -> Result<()> {
     ----- stdout -----
 
     ----- stderr -----
-    error: Found a virtual workspace root, but virtual projects do not support production dependencies (instead, use: `uv add --dev`)
+    error: Found a non-project workspace root; production dependencies are unsupported (instead, use: `uv add --dev`)
```

Comment:
Should we be suggesting that they make it a project? Should we just automatically migrate them to a project workspace root? (probably best as a follow-up if so)